js/test/date/testMeridiems.js

```diff
@@ -1896,11 +1896,11 @@ module.exports.testmeridiems = {
     },
     testMeridiem_pa_PK: function(test) {
         test.expect(3);
-        var fmt = DateFmt.getMeridiemsRange({locale:"pa-PK"});
+        var fmt = DateFmt.getMeridiemsRange({locale:"pa-Arab-PK"});
         test.ok(fmt !== null);
         
-        test.equal(fmt[0].name,  "ਪੂ.ਦੁ.");
-        test.equal(fmt[1].name,  "ਬਾ.ਦੁ.");
+        test.equal(fmt[0].name, 'AM');
```

Comment:
Also here, please make two tests: one with Arab, one with Guru.

Reply:
Same too. Don't worry We already have it